aws-opentelemetry-distro/src/amazon/opentelemetry/distro/exporter/otlp/aws/metrics/otlp_aws_emf_exporter.py

```diff
@@ -0,0 +1,482 @@
+                                "message": json.dumps(
+                                    self._create_emf_log(metric_records, resource_metrics.resource, timestamp_ms)
+                                ),
+                                "timestamp": timestamp_ms,
+                            }
+                        )
+
+            return MetricExportResult.SUCCESS
+        # pylint: disable=broad-exception-caught
+        # capture all types of exceptions to not interrupt the instrumented services
+        except Exception as error:
+            logger.error("Failed to export metrics: %s", error)
+            return MetricExportResult.FAILURE
+
+    def force_flush(self, timeout_millis: int = 10000) -> bool:
+        """
+        Force flush any pending metrics.
+
+        Args:
+            timeout_millis: Timeout in milliseconds
+
+        Returns:
+            True if successful, False otherwise
+        """
+        logger.debug("CloudWatchEMFExporter force flushes the buffered metrics")
+        return True
+
+    def shutdown(self, timeout_millis=None, **kwargs):
+        """
+        Shutdown the exporter.
+        Override to handle timeout and other keyword arguments, but do nothing.
+
+        Args:
+            timeout_millis: Ignored timeout in milliseconds
+            **kwargs: Ignored additional keyword arguments
+        """
+        # Intentionally do nothing
+        self.force_flush(timeout_millis)
+        logger.debug("CloudWatchEMFExporter shutdown called with timeout_millis=%s", timeout_millis)
+        return True
+
+
+def create_emf_exporter(
+    namespace: str = "OTelPython",
+    log_group_name: str = "/aws/otel/python",
+    log_stream_name: Optional[str] = None,
+    aws_region: Optional[str] = None,
+    debug: bool = False,
+    **kwargs,
+) -> CloudWatchEMFExporter:
+    """
+    Convenience function to create a CloudWatch EMF exporter with DELTA temporality.
+
+    Args:
+        namespace: CloudWatch namespace for metrics
+        log_group_name: CloudWatch log group name
+        log_stream_name: CloudWatch log stream name (auto-generated if None)
+        aws_region: AWS region (auto-detected if None)
+        debug: Whether to enable debug printing of EMF logs
+        **kwargs: Additional arguments passed to the CloudWatchEMFExporter
+
+    Returns:
+        Configured CloudWatchEMFExporter instance
+    """
+
+    # Set up temporality preference - always use DELTA for CloudWatch
+    temporality_dict = {
+        Counter: AggregationTemporality.DELTA,
+        Histogram: AggregationTemporality.DELTA,
+        ObservableCounter: AggregationTemporality.DELTA,
+        ObservableGauge: AggregationTemporality.DELTA,
+        ObservableUpDownCounter: AggregationTemporality.DELTA,
+        UpDownCounter: AggregationTemporality.DELTA,
+    }
+
+    # Configure logging if debug is enabled
+    if debug:
+        logging.basicConfig(level=logging.DEBUG)
+        logger.setLevel(logging.DEBUG)
```

Comment:
Will this enabling debug logs for just this EMF Exporter component, or for all python code in user's application?